CONTRIBUTING.md

```diff
@@ -173,11 +173,15 @@ in the future, they may be removed by the Repository Maintainer.
 Refactoring is a necessary part of any software project's evolution. The
 following guidelines cover refactoring pull requests for the project.
 
-There are three categories of refactoring, code only moves, code style fixes,
-code refactoring. In general refactoring pull requests should not mix these
-three kinds of activity in order to make refactoring pull requests easy to
-review and uncontroversial. In all cases, refactoring PRs must not change the
-behaviour of code within the pull request (bugs must be preserved as is).
+There are three categories of refactoring namely,
+1. Code only moves
+2. Code style fixes
+3. Code refactoring
+
+In general, refactoring pull requests should not mix these
+three kinds of activity. Following this guideline will make refactoring pull requests easy to
+review and uncontroversial. In all cases, refactoring pull requests must not change the
+behaviour of code. _Even bugs must be preserved as is._
```

Comment:
A sentence starting with `Even` looks a bit weird to me. Maybe move it back into brackets like it was before?

Reply:
updated the changes as suggested. thanks for the suggestions.